cargo-near/src/posthog_tracking.rs

```diff
@@ -1,49 +1,51 @@
-use base64::{engine::general_purpose, Engine as _};
 use reqwest::{header::HeaderMap, Client};
 use serde::Serialize;
 use std::{env, str};
 use tracing::debug;
+use rustc_version::version;
 
 const SEND_TRACKING_REQUEST_ERROR: &str = "Can't send tracking usage event";
 
 #[derive(Debug, Serialize)]
-struct MixpanelProperties {
-    token: String,
-    pkg_version: String,
+struct PosthogProperties {
+    language: String,
+    engine: String,
     os: String,
 }
 
 #[derive(Debug, Serialize)]
 struct TrackingData {
+    api_key: String,
     event: String,
-    properties: MixpanelProperties,
+    distinct_id: String,
+    properties: PosthogProperties,
 }
 
 pub(crate) fn track_usage() {
-    let properties = MixpanelProperties {
-        token: "24177ef1ec09ffea5cb6f68909c66a61".to_string(),
-        pkg_version: env!("CARGO_PKG_VERSION").to_string(),
+    let properties = PosthogProperties {
+        language: "rs".to_string(),
+        engine: version().unwrap().to_string(),
```

Comment:
There is no value in capturing Rust compiler version. We care about the cargo-near version, so please, see the previous version for CARGO_PKG_VERSION and remove rustc_version dependency

Reply:
Done